### PR TITLE
Add style for selected items in dropdown to match bootstrap-select

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
This PR adds the selected styling for dropdowns to match the styling set for bootstrap-select objects. Updated test pages to set the selected state on dropdowns where selection is applicable.

This fixes issue https://github.com/patternfly/angular-patternfly/issues/318

See the filter and sort dropdowns at http://rawgit.com/jeff-phillips-18/patternfly/fixes-dist/dist/tests/toolbar.html

@andresgalante @srambach

### CODE REVIEW

File: src/less/dropdowns.less [hunk 1]
```diff
@@ -78,6 +78,14 @@
     border-color: transparent;
   }
 }
+
+// Selected state
+.dropdown-menu > .selected > a {
```
Comment (andresgalante) [inline src/less/dropdowns.less:83]: Hi @jeff-phillips-18 Do we need the child selectors?

File: src/less/dropdowns.less [hunk 2]
```diff
@@ -78,6 +78,14 @@
...
+  background-color: @color-pf-blue !important;
```
Comment (andresgalante) [inline src/less/dropdowns.less]: also, do we need the importants?

File: src/less/dropdowns.less (hunk 1 above)
Comment (jeff-phillips-18) [reply src/less/dropdowns.less:83]: Yes, the structure is set such that selected is added to the li item and the style needs to be set on the a. This is similar to boostrap-select.less:
https://github.com/patternfly/patternfly/blob/master/src/less/bootstrap-select.less#L58

File: src/less/dropdowns.less (hunk 2 above)
Comment (jeff-phillips-18) [reply src/less/dropdowns.less]: I believe so. Again, this is very similar to bootstrap-select.less:

https://github.com/patternfly/patternfly/blob/master/src/less/bootstrap-select.less#L58

File: src/less/dropdowns.less (hunk 1 above)
Comment (andresgalante) [reply src/less/dropdowns.less:83]: right, ok then. We'll have to work on improving our selectors as we move forward with pf4.

Thanks Jeff 😄  you are awesome as always.

File: src/less/dropdowns.less (hunk 2 above)
Comment (andresgalante) [reply src/less/dropdowns.less]: Maybe we don't need them here and the specificity of the class is enough, can you try it without the importants?

**Comment by jeff-phillips-18** [comment]
I've updated to remove the !important and use the same color variables as bootstrap-select.

**Comment by andresgalante** [comment]
@jeff-phillips-18 great, thanks a lot 👍 

@srambach if its ok with you please merge it.

File: src/less/dropdowns.less [hunk 3]
```diff
@@ -78,6 +78,17 @@
     border-color: transparent;
   }
 }
+
+// Selected state
+.dropdown-menu > .selected > a {
+  background-color: @dropdown-link-active-bg;
+  border-color: @dropdown-link-active-border-color;
+  color: @color-pf-white;
+  small {
```
Comment (srambach) [inline src/less/dropdowns.less:87]: Pardon my ignorance - is this a media query? Why does the color change here? I'm probably missing it in the spec somewhere but wondered.

Comment (jeff-phillips-18) [reply src/less/dropdowns.less:87]: I can remove it. I only added it since it is there for bootstrap-select. Shall I?

Comment (andresgalante) [reply src/less/dropdowns.less:87]: @srambach thanks for the review. That's correct, the small text is faded. check it out here when you open the drop down:
https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/bootstrap-select.html
screen shot 2016-10-12 at 8 48 48 am

**Comment by andresgalante** [comment]
Thanks a lot @jeff-phillips-18 !

**Pull request merged**